dashboard/test/ui/test_status.js

```diff
@@ -422,3 +494,157 @@ function toggleHelpText() {
   }
 }
 helpLink.onclick = toggleHelpText;
+
+/**
+ * Changes the icon in your browser tab (replacing the favicon)
+ * to a dynamically-generated status icon of the given color
+ * and progress info.
+ * @param {string} [typeOrColor] Any valid HTML color string, or a
+ *   status string ('pass', 'fail', 'in-progress', 'not-started').
+ *   Defaults to gray (not started).
+ */
+function setTabStatusIcon(typeOrColor) {
```

Comment:
This function is imported from `dashboard/test/ui/js/set-tab-status-icon.js`, which is now deleted.

Why? Having a file in a sub-path made S3 upload messy, this wasn't a module we were ever going to re-use, its pretty short and is still well encapsulated, there's no build-system or webpack anyway, and it was the only file in dashboard/test/ui/js so it removed a folder.